src/runtime/basic-runtime.js

```diff
@@ -477,6 +484,11 @@ export class ConfiguredBasicRuntime {
     );
   }
 
+  /** Sets the callback when the offers flow is requested. */
+  setOnOffersFlowRequest(callback) {
```

Comment:
Similarly here, should this be a private method?

Reply:
Set it as private.